action.yml

```diff
@@ -78,8 +78,8 @@ runs:
       env:
         STEAM_HOME: ${{ steps.setup-steamcmd.outputs.directory }}
         STEAM_CMD: ${{ steps.setup-steamcmd.outputs.executable }}
-        username: ${{ inputs.username }}
-        password: ${{ inputs.password }}
+        steam_username: ${{ inputs.username }}
+        steam_password: ${{ inputs.password }}
```

Comment:
I didn't think it was a breaking change, since these values come from the inputs which haven't changed their name. The reason is it appeared somewhere in the Windows bash environment $username was getting set to empty string, because this is a variable name some shells automatically set to the current user or otherwise reserve.

Reply:
I see. Thanks for the explanation.

Reply:
Generally speaking we should be converting all these "env" to "with" variables. That way they'll be prepended with `INPUT_` which effectively solves this category of problem. But out of scope for this issue.